### PR TITLE
Add "collector troubleshooting" doc

### PR DESCRIPTION
**Pull request by smithclay**
This adds a short page that generally points users to the existing collector [troubleshooting docs](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md).

Two minor additions not covered in the main doc:
- Specifically calls out the "sending test data"/local exporter guidance (anecdotally a useful troubleshooting step for users doing first-time collector setup and not seeing data)
- Adds a high-level checklist of things to verify for complex setups (collectors to collectors, etc)

Think a future section that goes deeper into collector observability would be a great future addition.

---

Preview: https://deploy-preview-2382--opentelemetry.netlify.app/docs/collector/troubleshooting/

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
This is a great addition and starting point for troubleshooting, thanks!

I also wonder if we can crib some content from vendors about troubleshooting, too. Some of what's in [this doc](https://docs.honeycomb.io/getting-data-in/tracing/troubleshooting/) is also helpful, but not all of it. Anyways, happy to take this as-is and iterate on the topic as needed.

**Comment by smithclay** [comment]
@cartermp Agreed, there's good advice (and useful external tools like [otel-cli](https://github.com/equinix-labs/otel-cli) or https://github.com/fullstorydev/grpcurl and https://github.com/CtrlSpice/otel-desktop-viewer) scattered in many places.

For advice involving non-community but open-source tools, are we prohibited from linking?

**Comment by cartermp** [comment]
Off-hand I'd say no, definitely not prohibited. Vendor stuff is what's iffy since we can't favor one over another, even if it's done only because a vendor happens to have the best content for a topic :/

File: content/en/docs/collector/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: Troubleshooting
+weight: 32
+---
+
+The OpenTelemetry Collector supports a variety of metrics, logs, and extensions
+for troubleshooting collector health and performance.
+
+Detailed recommendations, including common problems, are detailed in the
+OpenTelemetry Collector GitHub repo's
+[troubleshooting document](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md).
```
Comment (svrnm) [inline content/en/docs/collector/troubleshooting.md:11]: I am wondering if this document should live on opentelemetry.io as well? cc @open-telemetry/collector-maintainers ?

**Comment by svrnm** [comment]
> Off-hand I'd say no, definitely not prohibited. Vendor stuff is what's iffy since we can't favor one over another, even if it's done only because a vendor happens to have the best content for a topic :/

We copied some content from vendor's in the past with their approval and adding them into the Acknowledgment section, so I see no issue with doing this once again.

File: content/en/docs/collector/troubleshooting.md (hunk 1 above)
Comment (chalin) [reply content/en/docs/collector/troubleshooting.md:11]: Yes! As is already mentioned in the following PR under "Collector":

- #833 

There is valuable information under [opentelemetry-collector/docs](https://github.com/open-telemetry/opentelemetry-collector/tree/main/docs), which (IMHO) would benefit from the increased visibility of being hosted directly on the OTel website.

File: content/en/docs/collector/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Troubleshooting
+weight: 32
```
Comment (chalin) [inline content/en/docs/collector/troubleshooting.md:3]:
```suggestion
weight: 25
```

The current weight leaves this page in a strange place:

> image

File: content/en/docs/collector/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: Troubleshooting
```
Comment (chalin) [inline content/en/docs/collector/troubleshooting.md:2]: Consider adding a short `description` field explaining what this page is about.

File: content/en/docs/collector/troubleshooting.md (hunk 1 above)
Comment (chalin) [inline content/en/docs/collector/troubleshooting.md:11]: I'd suggest moving the content of this paragraph (lines 9 to 11) to the end, under a section entitled (something like) **More**.

File: content/en/docs/collector/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+weight: 32
+---
+
+The OpenTelemetry Collector supports a variety of metrics, logs, and extensions
+for troubleshooting collector health and performance.
```
Comment (chalin) [inline content/en/docs/collector/troubleshooting.md:7]: The opening paragraph should let the reader know what they'll find in this page.

**Comment by svrnm** [comment]
I am happy to hav this, @smithclay can you give the suggestions from @chalin a look and resolve them, TY!

**Comment by smithclay** [comment]
@chalin Thanks for the feedback, have pushed the requested changes + rebased.

File: content/en/docs/collector/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+title: Troubleshooting
+description: Recommendations for troubleshooting the collector
+weight: 25
+---
+
+This document describes some options when troubleshooting the health or
+performance of The OpenTelemetry Collector. The Collector provides a variety of
+metrics, logs, and extensions for debugging issues.
```
Comment (chalin) [inline content/en/docs/collector/troubleshooting.md:9]:
```suggestion
This page describes some options when troubleshooting the health or
performance of the OpenTelemetry Collector. The Collector provides a variety of
metrics, logs, and extensions for debugging issues.
```

File: content/en/docs/collector/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+## Sending test data
+
+For certain types of issues, particularly verifing configuration and debugging
+network issues, it can be helpful to send a small amount of data to a collector
+configured to output to local logs. This is detailed in
+["Local exporters" in the collector configuration documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#local-exporters).
```
Comment (chalin) [inline content/en/docs/collector/troubleshooting.md:16]:
```suggestion
configured to output to local logs. For details, see
[Local exporters](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#local-exporters).
```

File: content/en/docs/collector/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+- How is the telemetry being exported from this component?
+- What format is the telemetry in?
+- How is the next hop configured?
+- Are there any network policies that prevent data from getting in or out?
+
+### More
+
+Detailed recommendations, including common problems, are in the OpenTelemetry
+Collector GitHub repo's
+[troubleshooting document](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md).
```
Comment (chalin) [inline content/en/docs/collector/troubleshooting.md:38]:
```suggestion
For detailed recommendations, including common problems, see 
[Troubleshooting](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md) from the Collector repo.
```

**Review by chalin — COMMENTED** [review]
Thanks. A few more copy edits (you'll probably have to rerun Prettier after accepting them), see inline. Otherwise LGTM.

**Review by chalin — APPROVED** [review]
LGTM, thanks!

**Pull request merged**